features/workflow.feature

```diff
@@ -140,3 +140,15 @@ Feature: Workflow
         And the "SIMPLE" job is not triggered
         And the "MASTER-PR" job is not triggered
         And that "REGEX" build uses the same SHA as the "STAGING" build
+
+    @ignore @workflow-chainPR
+    Scenario: chainPR
+        Given an existing pipeline on "master" branch with the workflow jobs:
```

Comment:
should we change this to `"main" branch`? I know GitHub defaults to `main` branch moving forward

I'm open to changing the repo default branch to "master" if you think that makes more sense